src/containers/bitset.c

```diff
@@ -117,9 +123,15 @@ bitset_container_t *bitset_container_clone(const bitset_container_t *src) {
     if (!bitset) {
         return NULL;
     }
-    // sizeof(__m256i) == 32
-    bitset->words = (uint64_t *)roaring_aligned_malloc(
-        32, sizeof(uint64_t) * BITSET_CONTAINER_SIZE_IN_WORDS);
+    if ( croaring_avx512() ) {
```

Comment:
Same comment here, I think that this needs to be guarded with CROARING_IS_X64.

Reply:
yes, i will add